src/components/feedback/FeedbackDirective.js

```diff
@@ -34,7 +34,7 @@ goog.require('ga_permalink');
             link: function(scope, element, attrs) {
               function validateSize(fileSize) {
                 if (fileSize > 10000000) { // 10 Mo
-                  alert($translate.instant('file_too_large') + ' (Max 10Mb)');
+                  alert($translate.instant('file_too_large') + ' (max. 10 Mo)');
```

Comment:
Please use english term when not translated. Mb is much more rocognized than Mo.